task6/src/components/movieDetail/movieDetail.js

```diff
@@ -0,0 +1,39 @@
+import React, { useContext } from 'react';
+import './movieDetail.scss';
+import LogoHeader from '../logoHeader/logoHeader';
+import { MovieContext } from '../../utils/contexts/movieContext';
+import {connect} from 'react-redux';
+
+function MovieDetail({ movies }) {
+  const {movieId} = useContext(MovieContext);
+  const selectedMovie = movies.find((movie) => movie.id === movieId);
```

Comment:
what if you deconstruct  the object here and then use only values  (poster,title,rate) in the jsx